### ISSUE TITLE
main.js - return json from "tarot.json" in getDeck() function

### ISSUE THREAD

**Issue by michaeljoelt** [michaeljoelt/tarot#2]
in main.js' getDeck() function:
replace the returned json with data from tarot.json (instead of storing it directly in main.js)

**Comment by Brandon-Schefstad** [comment]
I was able to import tarot.json by using a fetching it and a little restructuring. This is my first time attempting this so if there are better practices let me know.  displayCard() and flipCard() are unchanged except to make them async. 
`
// Loads data from tarot.json on click of draw button. Then passes data.cards to selectCard
// #####################################################################
async function loadLocalData() {
	await fetch('./tarot.json')
		.then((res) => res.json())
		.then((data) => {
			selectCard(data.cards);
		});
}
// Untouched
// #####################################################################
let isFront = 0;
document.querySelector('#draw').addEventListener('click', loadLocalData);

// Takes in deck object from line 7, selects a random integer <78 to locate an object in at that index. From this object we initialize the card's name and the url for its image.
// #####################################################################
async function selectCard(deck) {
	const randomNumber = Math.floor(Math.random() * 78);
	const card = deck[randomNumber];
	const card_name = card.name;
	const source = card.image;
	displayCard(card, card_name, source);
}
`

**Comment by michaeljoelt** [comment]
Hey hey! So I had a couple pull requests, and reviewed [VeitchM](https://github.com/VeitchM)'s changes  first and merged to main; and then was reviewing yours and realized it was for the same issue! Dohhh.... Sorry about that! I'm also getting more used to the collaborative side of github. 

I've updated the README's collaboration notes to request folks leave a comment on an issue before starting work on it. Hopefully that will help!

It looks like VeitchM handled this by pulling in the data from tarot.json at the beginning of main.js, and storing it in variable "simplytarot", leaving the rest of code unchanged. so basically just:
```
let simplytarot; 
getDeck();

async function getDeck() {
    await fetch('tarot.json').then(response => response.json()).then( (data) => {simplytarot =  data}  )
}
```
Similar to your changes but didn't require altering functions beyond getDeck() so thought you might find it interesting. If there's any changes you think are important that you made and I overlooked, let me know and I can open a new issue.

I am going clean up my main.js code and include better comments to help make it easier to collaborate further on. especially as we try to explore the filtering functionality and other enhancements. 

Thanks for digging in to this project with us!
